src/roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -40,9 +40,9 @@ def temporary_deprecation_message():
 
     raise NotImplementedError(
         "ROIExtractors no longer supports direct write to NWB. This method will be removed in a future release.\n\n"
-        "Please install nwb-conversion-tools and import the corresponding write method from there.\n\nFor example,\n\n"
+        "Please install neuroconv and import the corresponding write method from there.\n\nFor example,\n\n"
         "from roiextractors import NwbSegmentationExtractor\nNwbSegmentationExtractor.write_segmentation(...)\n\n"
-        "would become\n\nfrom nwb_conversion_tools import roiextractors\nroiextractors.write_segmentation(...)"
+        "would become\n\nfrom neuroconv import roiextractors\nroiextractors.write_segmentation(...)"
```

Comment:
This would not work though, I would just replace lines 43-45 with
```suggestion

```
"""
Please install neuroconv and import the corresponding segmentation interface from there. For example, <import an example segmentation interface and call .add_to_nwbfile>.
"""